presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -70,7 +72,7 @@ public void testAcos()
     {
         for (double doubleValue : DOUBLE_VALUES) {
             assertFunction("acos(" + doubleValue + ")", DOUBLE, Math.acos(doubleValue));
-            assertFunction("acos(FLOAT '" + (float) doubleValue + "')", DOUBLE, Math.acos((float) doubleValue));
+            assertFunction("acos(FLOAT '" + doubleValue + "')", DOUBLE, Math.acos((float) doubleValue));
```

Comment:
You mean add acos that accepts float? We need to have a discussion about this on Monday. I briefly spoke to @maciejgrzybek about this. From what I see on Postgres it only accepts Double precision and returns Double precision. Need to check what is the standard for this is, last time when we did something similar we followed the types for functions supported by Postgres